### DISCUSSION TITLE
全新空洞被发现

### DISCUSSION THREAD

**Discussion by xiaosheng31** [share121/inter-knot#3029] [其他]
空洞求生指南
……接上信号了？不错，能看到这个网站就代表着你在这里存活下来的概率又高了一些，而且你是一个在这种奇怪地方都有心思再看一眼手机的人，相信你的意志力和镇静足以让你能遵循我们的指示顺利与其他流浪者安全会和。现代人手机不离身倒的确有点道理。

首先，不要惊慌，你目前所处的地方没有多少威胁，你自己内心的恐惧和孤寂感才是最大的威胁，但请尽快离开这里吧——这里没有水，没有食物，尽管有其他流浪者但你几乎不可能见到，只有泛黄的墙纸，潮湿的地毯和嗡嗡作响的日光灯。你在看到这个网站后应该有足够的时间来尝试离开这里.什么你怎么进来的我可以告诉你进来的人来自四面八方根据可靠的研究后室空间可能联通所有地方
总之欢迎来到后室，一个脱离于我们正常的现实世界的地方我们在这里被称之为流浪者不同于你们新艾利都的市民以及你们哪里的绳匠。你所目前所处的地方，被称之为[Level 0]。这是一般情况下像你这样不小心脱离了现实，或者说，“[卡出]了现实世界”的人来到后室的第一站。后室就是一个由不同空间，我们一般称之为“[楼层]，所组成的迷宫，每个楼层都有它自己奇怪的特性，并以一种诡异而复杂的方式相互连接在一起。有些楼层是相对安全的，比如你现在所在的Level 0，和接下来你会来到的[Level 1]，另一些楼层则是危险的，程度各不相同，但几乎每个楼层都会有威胁存在，这里可不如你以前所在的现实世界安全，连物理定律都可能被篡改，还有各种奇形怪状的，我们称之为“[实体]的后室生物存在。在这里需要时时保持警惕，否则一不小心就会没命【虽然有时在后室的“死亡”都和常理不太一样，但还是谨慎为妙，死后的世界我想你是不会想要亲身见识的。】
不要想着回去因为你出去的几率微乎其微
你说回到现实？没门。目前尚未有任何可以从后室回到现实的方式被证实，也不存在任何从后室回到现实者的可靠记录。接受这一切吧——此处就是你的新家。祝你好运。
根据大致研究后室大致有9层但有不同的层级
常规楼层有多种规模，安全程度和勘探程度，是[后室]的蓝图。有些楼层能作为长期运转的社区的选址，而有些则极其危险。许多楼层都有一种“阈限空间”感，让身处其中的人发毛。在浏览一些楼层的图片时，也许你会兴奋地指着屏幕大叫“这里我熟！”但我们必须警告你[后室]并不遵循地球的法则⸺连物理定律都被抛开⸺这里面满是异常与混乱。

我们敦促你不要在任何单一的楼层里徘徊，除非有文件证明其安全性。楼层可能随着时间的推移而改变，甚至消失⸺每当这种情况发生时，一个完全不同的空间将取代旧的楼层，先前的楼层将不再存在于这个数据库中，但可以在我们的档案中找到。请记住，数字是任意的⸺在大多数情况下，楼层的数字与楼层的属性没有关联。当你在我们的数据库中创建一个新的楼层页面时，请牢记这一点。
最后再次祝你好运

![Uploading screenshot-1724042828065.png…]()

你要明白，你的责任是创造一份描述详尽且沉浸感高的文件，让穿越你所记录的楼层的读者和你一样了解这个楼层。记住，你在如何描述你的发现方面有创作自由⸺问问你自己，一个迷路的流浪者在看到你的楼层条目时，能否利用你的信息在这个楼层中找到正确的方向。如果你缺乏信息，不要担心；另一个流浪者总是可以填补的。然而，请确保你对你所记录的楼层有一个大致的了解，并有足够的阐述来使流浪者的旅行更容易。

**Comment by RGthx** [comment]
空洞——但是后室是吧
绳匠呢？救一下啊绳匠

**Reply by xiaosheng31** [reply]
目前新空洞研究资料不足贸然营救只会付出巨大伤亡

**Comment by lllliufeng** [comment]
**信号丢失**
